src/containers/StructurePage/resourceComponents/AllResourcesGroup.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import { Plus } from '@ndla/icons/action';
+import BEMHelper from 'react-bem-helper';
+import AddTopicResourceButton from './AddTopicResourceButton';
+import Accordion from '../../../components/Accordion';
+import ResourceItems from './ResourceItems';
+import AddResourceModal from './AddResourceModal';
+import { RESOURCE_TYPE_LEARNING_PATH } from '../../../constants';
+
+import { Filter } from '../../../interfaces';
+
+export const classes = new BEMHelper({
+  name: 'topic-resource',
+  prefix: 'c-',
+});
+
+interface Props {
+  topicResources: {
+    resources: [];
+    contentType: string;
+  }[];
```

Comment:
For meg ser det ut som om topicResources er en array av Resource. ikkje et objekt med en array.
```suggestion
  topicResources: Resource[];
```